src/services/resource-creator.js

```diff
@@ -1,60 +1,57 @@
-'use strict';
-var _ = require('lodash');
-var P = require('bluebird');
-var ErrorHTTP422 = require('./errors').ErrorHTTP422;
-var Interface = require('forest-express');
-var ResourceGetter = require('./resource-getter');
-var CompositeKeysManager = require('./composite-keys-manager');
+const _ = require('lodash');
+const P = require('bluebird');
+const { ErrorHTTP422 } = require('./errors');
```

Comment:
Require "dependencies" before "local code" (airbnb eslint - we might as well do that with the new code we write :) ) 
```
const Interface = require('forest-express');
const { ErrorHTTP422 } = require('./errors');
```

Reply:
Good catch 🙏 ! Fixed!